ui/components/ui/entities/get-provider-logo.tsx

```diff
@@ -7,8 +7,7 @@ import {
   KS8ProviderBadge,
   M365ProviderBadge,
 } from "@/components/icons/providers-badge";
-
-export type ProviderType = "aws" | "azure" | "gcp" | "kubernetes" | "m365";
```

Comment:
Could you please revert this change? I believe it's not part of the Lighthouse scope, thank you!

For more context: I'm already moving this in another PR. We try to keep the scope of each PR as limited as possible, so it's clear exactly what was done and where.

Reply:
Same as https://github.com/prowler-cloud/prowler/pull/7508#discussion_r2068437608